src/components/UserDashboard/WishlistPage.tsx

```diff
@@ -1,138 +1,138 @@
-// Dependencies
-import { useState, ChangeEvent, useEffect } from 'react';
-import { FiShare } from 'react-icons/fi';
-import styled from '@emotion/styled';
-import { FaHeart } from 'react-icons/fa';
-import { useParams } from 'react-router-dom';
+// // Dependencies
+// import { useState, ChangeEvent, useEffect } from 'react';
+// import { FiShare } from 'react-icons/fi';
+// import styled from '@emotion/styled';
+// import { FaHeart } from 'react-icons/fa';
+// import { useParams } from 'react-router-dom';
 
-// Components
-import AmiiboItem from './AmiiboItem';
-import Popup from './WishlistPopup';
-import { Amiibo } from '../../types/Amiibo';
-import { User } from '../../types/User';
-import { getUser } from '../../features/user/userAPI';
-import { removeFromWishlist } from '../../features/user/updateWishlist';
+// // Components
+// import AmiiboItem from './AmiiboItem';
+// import Popup from './WishlistPopup';
+// import { Amiibo } from '../../types/Amiibo';
+// import { User } from '../../types/User';
+// import { getUser } from '../../features/user/userAPI';
+// import { removeFromWishlist } from '../../features/user/updateWishlist';
```

Comment:
I think this might be where the main error is coming from. 
The name is correct but the location of the ```removeFromWishlist``` function has been moved to userAPI.ts.